bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -506,6 +524,9 @@ ldap_initialize() {
         if [[ -f "$LDAP_CUSTOM_SCHEMA_FILE" ]]; then
             ldap_add_custom_schema
         fi
+        if ! is_dir_empty "$LDAP_CUSTOM_SCHEMA_DIR"; then
+            ldap_add_custom_schemas
```

Comment:
Can you fix the indentation, please?

Reply:
As above.

Reply:
I've checked it again and it's already in the correct format.